source/agora/test/ManyValidators.d

```diff
@@ -65,16 +65,16 @@ unittest
         .array;
 
     // 16 utxos for freezing, 8 utxos for creating a block later
-    txs ~= spendable[6].split(WK.Keys.byRange.take(16).map!(k => k.address)).sign();
+    txs ~= spendable[6].split(WK.Keys.byRange.take(conf.outsider_validators + GenesisValidators).map!(k => k.address)).sign();
     txs ~= spendable[7].split(WK.Keys.Genesis.address.repeat(8)).sign();
     txs.each!(tx => nodes[0].putTransaction(tx));
-    // block 8
-    network.expectBlock(Height(8));
+    // block 17
+    network.expectBlock(expected_block++);
```

Comment:
Ditto please move it outside to make it clearer.

Reply:
We do this check so many times I thought this keeps it cleaner.